package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -0,0 +1,652 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+"""\
+Low level DCD  trajectory reading - :mod:`MDAnalysis.lib.formats.libdcd`
+------------------------------------------------------------------------
+
+:mod:`libdcd` contains the class :class:`DCDFile` to read and write frames of a
+DCD file. The class tries to behave similar to a normal file object.
+
+:mod:`libdcd` contains the classes :class:`XTCFile` and
+:class:`TRRFile`. Both can be used to read and write frames from and to
+Gromacs_ XTC and TRR files. These classes are used internally by MDAnalysis in
+:mod:`MDAnalysis.coordinates.XTC` and :mod:`MDAnalysis.coordinates.TRR`. They
+behave similar to normal file objects.
+
+For example, one can use a :class:`DCDFile` to directly calculate mean
+coordinates (where the coordinates are stored in `x` attribute of the
+:class:`namedtuple` `frame`):
+
+.. code-block:: python
+   :emphasize-lines: 1,2,5
+
+   with DCDFile("trajectory.dcd") as dcd:
+       header = dcd.header
+       mean = np.zeros((header['natoms'], 3))
+       # iterate over trajectory
+       for frame in dcd:
+           mean += frame.x
+   mean /= header['natoms']
+
+
+Besides iteration one can also seek to arbitrary frames using the
+:meth:`~DCDFile.seek` method.
+
```

Comment:
https://github.com/MDAnalysis/mdanalysis/pull/1372#issuecomment-308179281 (apparently, email replies are not added to a comment thread)

[`catdcd`](http://www.ks.uiuc.edu/Development/MDTools/catdcd/) is also acknowledged in `AUTHORS`.